.github/workflows/main.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             win64,
             mingw32,
             mingw64,
-            windows-2019,
+            windows-2022,
```

Comment:
Could you please update the dependency of test-pass job at the end of the workflow?

Updating it would fix the pipeline

Reply:
Sorry my brain wasn't working, this obviously isn't an issue